components/docs-chef-io/content/automate/ha_backup_restore_aws_efs.md

```diff
@@ -27,7 +27,7 @@ gh_repo = "automate"
 
 A shared file system is always required to create **OpenSearch** snapshots. To register the snapshot repository using OpenSearch, it is necessary to mount the same shared filesystem to the exact location on all master and data nodes. Register the location in the `path.repo` setting on all master and data nodes.
 
-### Setting up the backup configuration
+### Setting up the Backup Configuration
```

Comment:
Headings should be sentence case. Use this if you're not sure. It's generally right --> https://titlecaseconverter.com/sentence-case/